src/Message/CreatePaymentMethodRequest.php

```diff
@@ -266,6 +271,27 @@ public function setUpdateSubscriptions($value)
         return $this->setParameter('updateSubscriptions', $value);
     }
 
+    /**
+     * If set to false we set the payment method active flag to be false
+     *
+     * @return null|bool
+     */
+    public function getActivatePaymentMethod()
+    {
+        return $this->getParameter('activate');
```

Comment:
can you make the parameter 'active' instead of 'activate'?

`setActivate(false)` doesn’t really make sense linguistically--it’s not clear that that would deactivate something
whereas `setActive(false)` makes it clear that the payment method will not be active